Makefile

```diff
@@ -11,6 +11,11 @@ SERVICE_NAME=nightfall_dlp
 BINARY_NAME=./$(SERVICE_NAME)
 GO_TEST_ENV?=test
 
+NAME=nightfallai/nightfall-dlp
```

Comment:
should this be an underscore?

Reply:
I created the docker hub repo with the dash but will update to use underscore, since probably makes more sense